src/hooks/Orderbook/useDrawOrderbook.ts

```diff
@@ -103,28 +103,26 @@ export const useDrawOrderbook = ({
   }, [canvas]);
 
   const drawBars = ({
-    barType,
     ctx,
-    depthOrSizeValue,
-    gradientMultiplier,
+    value,
+    gradientMultiplier = 1.3,
```

Comment:
[nit] maybe extract to a const and add a comment/link to this PR so people know this is an arbitrary number, in case in the future we want to change this number